### DISCUSSION TITLE
Welcome to Bee Discussions!

### DISCUSSION THREAD

**Discussion by acud** [ethersphere/bee#2330] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what excites you in the Swarm ecosystem!

**Comment by ldeffenb** [comment]
Greetings all!  As many of you know, I'm interested in the use of swarm as an alternate means of storing, distributing, and retrieving OpenStreetMap rendered tiles for mapping purposes instead of the current centralized tile server.  I'm currently working on putting a pre-rendered tile set into the swarm as a single collection making a fully self-contained and browsable map.

**Comment by lucille-bellepleure** [comment]
Hi! I'm Michelle Plur and I'm working for Swarm as Comms Lead. Also I have been working on some prototype dapps that use Swarm for storage (check my github profile). I am looking forward to engaging in this new Discussion feature! X

**Comment by bertonhe** [comment]
will we have a roadmap about how to attract more users to use our node for storage!

**Comment by mfw78** [comment]
Hi, for the incentive structure for bee, can we please have it calculated what the precise timesettlement bandwidth allowance is? Additionally is there any provision in the inter-bee communication to communicate what the free bandwidth allowance is, and therefore allow bee to be biased towards the cheapest method of upload / download (who gives out the most free bandwidth), but also enable a means of control for bee operators over the timesettlement allowance?
